src/pocketmine/Server.php

```diff
@@ -2935,7 +2936,8 @@ private function registerEntities(){
 		Entity::registerEntity(Egg::class);
 		Entity::registerEntity(Enderman::class);
 		Entity::registerEntity(FallingSand::class);
-		Entity::registerEntity(FishingHook::class);
+        Entity::registerEntity(FishingHook::class);
+        Entity::registerEntity(FloatingText::class);
```

Comment:
I'm using PhpStorm, and just create a new line, and it is the right distance. Even if you look it in GitHub, the distance is right. Blame PhpStorm if it's wrong.